packages/storybook/.storybook/main.js

```diff
@@ -86,7 +86,7 @@ module.exports = {
         '@material-ui/x-grid': path.resolve(__dirname, '../../../packages/grid/x-grid/src'),
         '@material-ui/x-license': path.resolve(__dirname, '../../../packages/license/src'),
       },
-      modules: [path.join(__dirname, '../../../'), 'node_modules'],
+      // modules: [path.join(__dirname, '../../../'), 'node_modules'],
```

Comment:
To test the build locally? Don't forget to uncomment before merging.

Reply:
Just tested and everything works fine without this.
I suspect it is the cause of your issue with the compilation of SB

Reply:
Yeah works great without.

```suggestion
```